packages/frontend/src/pages/Flows/index.tsx

```diff
@@ -24,159 +20,113 @@ import EmptyFlows from './components/EmptyFlows'
+  const { isOpen, onOpen, onClose } = useDisclosure()
 
   const { data, loading } = useQuery(GET_FLOWS, {
     variables: {
       ...getLimitAndOffset(page),
-      name: flowName,
+      name: input,
     },
   })
 
   const { pageInfo, edges } = data?.getFlows || {}
-  const flows: IFlow[] = edges?.map(({ node }: { node: IFlow }) => node)
-  const hasFlows = flows?.length > 0
-  const isSearching = flowName !== '' || page !== 1
-
-  const isEmptyState = !hasFlows && !isSearching
-  const isEmptySearchResults = !loading && !hasFlows && isSearching
-  const hasPagination = pageInfo && pageInfo.totalCount > FLOW_PER_PAGE
+  const flows: IFlow[] = edges?.map(({ node }: { node: IFlow }) => node) ?? []
+  const hasPagination =
+    !loading && pageInfo && pageInfo.totalCount > FLOW_PER_PAGE
+  const hasNoUserFlows = flows.length === 0 && !isSearching
 
   return (
-    <>
-      <Container py={9}>
-        {!isEmptyState && (
-          <Grid
-            templateAreas={{
-              base: `
-              "title button"
-              "search search"
-            `,
-              md: `"title search button"`,
-            }}
-            gridTemplateColumns={{ base: '1fr auto', md: '2fr 1fr auto' }}
-            columnGap={3}
-            rowGap={5}
-            alignItems="center"
-            pl={{ base: '0', md: '2rem' }}
-            mb={6}
-          >
-            <GridItem area="title">
-              <PageTitle title={FLOWS_TITLE} />
-            </GridItem>
-
-            <GridItem area="search">
-              <SearchInput
-                searchValue={flowName}
-                onChange={onSearchInputChange}
-              />
-            </GridItem>
-
-            <GridItem area="button">
-              <ConditionalIconButton
-                type="submit"
-                size="lg"
-                icon={<BiPlus />}
-                data-test="create-flow-button"
-                onClick={onOpen}
-              >
-                Create Pipe
-              </ConditionalIconButton>
-            </GridItem>
-          </Grid>
-        )}
-
-        <ApproveTransfersInfobox />
-
-        {loading && (
-          <Center mt={8}>
-            <PrimarySpinner fontSize="4xl" />
-          </Center>
-        )}
-
-        {!loading && isEmptyState && <EmptyFlows onCreate={onOpen} />}
-
-        {!loading &&
-          flows?.map((flow) => <FlowRow key={flow.id} flow={flow} />)}
-
-        {isEmptySearchResults && (
-          <NoResultFound
-            description="We couldn't find anything"
-            action="Try using different keywords or checking for typos."
-          />
-        )}
-
-        {hasPagination && (
-          <Flex justifyContent="center" mt={6}>
-            <Pagination
-              currentPage={pageInfo?.currentPage}
-              onPageChange={handlePageChange}
-              pageSize={FLOW_PER_PAGE}
-              totalCount={pageInfo?.totalCount}
+    <Container py={9} overflowY={'hidden'}>
```

Comment:
<img width="1486" alt="image" src="https://github.com/user-attachments/assets/d3819af5-d80a-468e-a083-5e9bcdbf130f">

can i check why is the `overflowY` added ah, cus now with one pipe only the context menu is hidden leh